app/packages/pyodide-language-server/README.rst

```diff
@@ -0,0 +1,161 @@
+Python Language Server
+======================
```

Comment:
This me vendoring in the Python Language Server. It needs some tweaking to work with Pyodide, but once complete and hooked up, the VS Code editor within the browser will get all the completion/hints magic.